src/test/java/dev/openfeature/sdk/FlagEvaluationSpecTest.java

```diff
@@ -82,12 +78,12 @@ void getApiInstance() {
     @Test void providerAndWait() {
         FeatureProvider provider = new TestEventsProvider(500);
         OpenFeatureAPI.getInstance().setProviderAndWait(provider);
-        assertThat(api.getProvider().getState()).isEqualTo(ProviderState.READY);
+        assertThat(api.getProviderState()).isEqualTo(ProviderState.READY);
```

Comment:
We can instead get a client (even before we set the provider) and then check the Provider state there, so we can remove this `getProviderState`.

Reply:
Done: https://github.com/open-feature/java-sdk/pull/1096/commits/45e9ba2e0b33ee84b0449003daff3f03cb820a45